src/cli/info.ts

```diff
@@ -4,10 +4,10 @@ import Debug from 'debug';
 import { createRequire } from 'module';
 import type { CommandBuilder } from 'yargs';
 
+import packageJson from '../../package.json';
```

Comment:
I haven't tested it, but does it work right now? Shouldn't it be:
```suggestion
import packageJson from '../../package.json' assert { type: 'json' };
```

Reply:
@mmiszy you're too quick! :) before you commented there's a new commit to disable it